src/PerimeterxFieldExtractor.php

```diff
@@ -11,26 +11,28 @@ class PerimeterxFieldExtractor {
     private $contentType;
     private $encoding;
     private $extractionFields;
+    private $callbackName;
```

Comment:
callbackName? is that the name of the callback or the callback itself (a reference to the callback is the callback)

Reply:
It's not a reference -- it's the string of the name of the callback.